app/assets/stylesheets/_shirts.scss

```diff
@@ -0,0 +1,21 @@
+div.shirt-banner {
+  margin-bottom: $base-spacing;
+  overflow-x: hidden;
+  white-space: nowrap;
+
+  a {
+    @include animation(marquee 20.0s linear infinite);
+    color: white;
+    display: inline-block;
+    font-style: italic;
+    font-weight: bold;
+    padding-left: 100%;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+}
+
+@include keyframes(marquee) {
+  0% { @include transform(translate(0, 0)); }
+  100% { @include transform(translate(-100%, 0)); }
+}
```

Comment:
From those docs:
>**Obsolete**
>This feature is obsolete. Although it may still work in some browsers, its use is discouraged since it could be removed at any time. Try to avoid using it.